crates/ruff_python_formatter/tests/snapshots/black_compatibility@simple_cases__comments4.py.snap

```diff
@@ -122,11 +122,10 @@ def foo3(list_a, list_b):
 -        .order_by(User.created_at.desc())
 -        .with_for_update(key_share=True)
 -        .all()
--    )
-+        filter
-+    )(db.not_(User.is_pending.astext.cast(db.Boolean).is_(True))).order_by(
-+        User.created_at.desc()
-+    ).with_for_update(key_share=True).all()
++        filter(db.not_(User.is_pending.astext.cast(db.Boolean).is_(True))).order_by(
```

Comment:
Requires call chain formatting

Reply:
yep i had opened https://github.com/astral-sh/ruff/issues/5343 to track this